src/components/BooksQuery/BooksQuery.tsx

```diff
@@ -48,7 +72,7 @@ const BooksQuery = () => {
         <form onSubmit={handleSubmit}>
           <input
             type="text"
-            value={bookName}
+            value={bookName ? bookName : searchInput}
```

Comment:
Not needed.

 You should just use the component state (bookName)

AND IN CASE you need to do something like this, a more consise way is this

```tsx
bookName || searchInput
```

instead of

```tsx
bookName ? bookName : searchInput
```